src/integrationTest/java/org/opensearch/security/api/SslCertsRestApiIntegrationTest.java

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
shouldn't the package for this test be: `src/test/java/org/opensearch/security/dlic/rest/api/` ?

Reply:
I was thinking about it as well. And decided to stop on `api`. There are 2 reasons:
- new tests we added to api
- it is difficult for new developer to find tests we use t test REST API  :-)

Reply:
i forget, what is the dlic for here? I feel like that is what might be confusing to me.

Reply:
Same. I do not know.